src/bojack/cli.cr

```diff
@@ -42,13 +40,30 @@ cli = Commander::Command.new do |command|
     end
 
     command.flags.add do |flag|
+      flag.name = "log"
+      flag.long = "--log"
+      flag.default = ""
+      flag.description = "Log output file."
+    end
+
+    command.flags.add do |flag|
       flag.name = "log-level"
       flag.long = "--log-level"
       flag.default = 1
       flag.description = "Log severity."
     end
 
     command.run do |options, arguments|
+      output = if options.string["log"].empty?
+                 STDOUT
```

Comment:
I agree with you, but that's the Crystal standard, I'm using https://github.com/crystal-lang/sublime-crystal